packages/cardano-services-client/package.json

```diff
@@ -5,18 +5,29 @@
   "engines": {
     "node": "^14"
   },
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
   "repository": "https://github.com/input-output-hk/cardano-js-sdk/packages/cardano-services-client",
   "contributors": [
     "Rhys Bartels-Waller <rhys.bartelswaller@iohk.io> (https://iohk.io)",
     "Martynas Kazlauskas <martynas.kazlauskas@iohk.io> (https://iohk.io)"
   ],
   "license": "MPL-2.0",
   "scripts": {
-    "build": "tsc --build ./src",
-    "generate": "graphql-codegen",
+    "build:esm": "tsc -p src/tsconfig.json --outDir ./dist/esm --module es2020",
+    "build:cjs": "tsc --build src",
+    "build": "run-s build:cjs build:esm module-fixup",
+    "module-fixup": "shx cp ../../build/cjs-package.json ./dist/cjs/package.json && cp ../../build/esm-package.json ./dist/esm/package.json",
```

Comment:
It's needed for node.js es module resolution. Main `package.json` files of packages do not specify `type` because we want them to work in both commonjs and esm setups. A separate `package.json` under `dist/esm` is required to treat files under it as es modules. The `cjs-package.json` one might actually be redundant because commonjs is the default `type`, but I put it in there for consistency and future-proofing.